src/Orleans/Logging/LoggerWrapper.cs

```diff
@@ -0,0 +1,183 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Orleans.Runtime
+{
+    //TODO: Mark it as [Obsolete] after all runtime has migrated
+    internal class LoggerWrapper : Logger
```

Comment:
Shouldn't this be `LoggerImpl` as it was before? It would make following its history (and diff) easier

Reply:
LoggerWrapper is a wrapper on top of Microsoft.Extensions.Logging.ILogger, which doesn't share similarity with LoggerImpl. So I cannot see why it help its history. 
And I cannot make it as LoggerImpl as least now, since it is part of the old logging infra, which the remaining code base is still using. Until I completely migrate the runtime to new asp.net abstraction, LoggerImpl will stay as is. That is my current plan